packages/server-wallet/src/handlers/__test__/join-channel.test.ts

```diff
@@ -0,0 +1,31 @@
+import matchers from '@pacote/jest-either';
```

Comment:
Didn't see this in the PR, was this installed to dev dependencies?

Reply:
It's already a dev dependency!